src/main/java/net/minecraftforge/common/loot/LootModifierManager.java

```diff
@@ -0,0 +1,80 @@
+package net.minecraftforge.common.loot;
+
+import java.util.Collection;
+import java.util.HashMap;
+//import static net.minecraft.client.resources.JsonReloadListener.LOGGER;
+
+import java.util.Map;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.ImmutableMap.Builder;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonObject;
+
+import net.minecraft.client.resources.JsonReloadListener;
+import net.minecraft.profiler.IProfiler;
+import net.minecraft.resources.IResourceManager;
+import net.minecraft.server.MinecraftServer;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.world.storage.loot.conditions.ILootCondition;
+import net.minecraft.world.storage.loot.conditions.LootConditionManager;
+import net.minecraft.world.storage.loot.functions.ILootFunction;
+import net.minecraft.world.storage.loot.functions.LootFunctionManager;
+import net.minecraftforge.fml.LogicalSide;
+import net.minecraftforge.fml.common.thread.EffectiveSide;
+import net.minecraftforge.registries.ForgeRegistries;
+
+public class LootModifierManager extends JsonReloadListener {
+    private static LootModifierManager instance;
+    public static final Logger LOGGER = LogManager.getLogger();
+    private static final Gson GSON_INSTANCE = (new GsonBuilder()).registerTypeHierarchyAdapter(ILootFunction.class, new LootFunctionManager.Serializer()).registerTypeHierarchyAdapter(ILootCondition.class, new LootConditionManager.Serializer()).create();
+
+    private Map<ResourceLocation, IGlobalLootModifier> registeredLootModifiers = ImmutableMap.of();
+
+    public LootModifierManager() {
+        super(GSON_INSTANCE, "loot_modifiers");
+        instance = this;
+    }
+
+    public static LootModifierManager getInstance() {
+        return instance;
+    }
+
+    @Override
+    protected void apply(Map<ResourceLocation, JsonObject> splashList, IResourceManager resourceManagerIn, IProfiler profilerIn) {
```

Comment:
This is a custom class, which means we don't have to use the same parameter names. So... use real names here... splashList is DEFINITLY not a list of splashes